moorhuhn-service/src/main/resources/application.properties

```diff
@@ -0,0 +1,13 @@
+spring.main.banner-mode=off
+
+spring.jpa.hibernate.ddl-auto=none
+
+spring.sql.init.mode=always
+spring.sql.init.platform=postgres
+spring.datasource.url=jdbc:postgresql://localhost:5432/test
+spring.datasource.username=postgres
+spring.datasource.password=asdf
```

Comment:
Did... did you just commit a DB password?
And something as insecure as that?

The normal approach to that is to create an extra `.properties` file that is explicitly ignored in the `.gitignore`.
You can then notify Spring to use this extra file by calling
`@PropertySource(value = classpath:db.properties")` in your main/ configuration class.

This is one of the rare instances where I recommend a force push to get rid of the password.